exchanges/graphcache/src/store/store.ts

```diff
@@ -154,11 +155,39 @@ export class Store implements Cache {
     invalidate(this, createRequest(query, variables));
   }
 
+  invalidateEntity(entity: Data | string) {
```

Comment:
Is it worth calling this just “invalidate” or is that a little misleading? 🤠

Reply:
I don't mind calling it invalidate if we're planning to deprecate `invalidateQuery`